src/main/resources/application.properties

```diff
@@ -1 +1,4 @@
-
+spring.jpa.hibernate.ddl-auto=create
+#spring.datasource.url=jdbc:mysql://localhost:3306/
+#spring.datasource.username=
+#spring.datasource.password=
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.